topology/pre-processor.c

```diff
@@ -485,9 +485,12 @@ static int pre_process_include_conf(struct tplg_pre_processor *tplg_pp, snd_conf
 		snd_input_t *in;
 		snd_config_t *n;
 		regex_t regex;
-		const char *filename;
```

Comment:
@aiChaoSONG the code looks good. Can you please add an example of this in the commit message and what the limitations are?